src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -8,12 +8,14 @@
 
   &__search-input {
     text-align: left;
-    font-size: 0.9em;
+    font-size: 12px;
+    height: 32px;
   }
 
   &__clear-search {
     cursor: pointer;
     opacity: 0.4;
+    width: 24px;
```

Comment:
Magic numbers. `$spacing--*` or custom `$variables` could be used.

Reply:
fixed 61d9df7